scripts/integration

```diff
@@ -1,36 +1,16 @@
 #!/bin/bash
 set -ex
 function echo_with_time {
-    echo "$(date --utc +%Y-%m-%dT%H:%M:%SZ) "$@""
+    echo "$(date --utc +%Y-%m-%dT%H:%M:%SZ) $*"
 }
 
-source $(dirname $0)/version
+source ./scripts/version
```

Comment:
I thought leaving it as `$(dirname $0)` would be better in case you're running this from some other directory